src/api/.rubocop.yml

```diff
@@ -141,6 +142,9 @@ Style/TernaryParentheses:
 Style/ZeroLengthPredicate:
   Enabled: true
 
+Style/FrozenStringLiteralComment:
```

Comment:
why are you disabling this cop and how is this related to the update? :thinking:

Reply:
I have no idea whats going on with rubocop in this PR... It was telling me to add this comment to a file:
`# frozen_string_literal: true`

But that also made some specs fail too...